src/app/components/client/ExposureCard.tsx

```diff
@@ -45,15 +45,44 @@ export type ExposureCardProps = {
 };
 
 export const ExposureCard = ({ exposureData, ...props }: ExposureCardProps) => {
-  return isScanResult(exposureData) ? (
-    <ScanResultCard {...props} scanResult={exposureData} />
-  ) : (
-    <SubscriberBreachCard {...props} subscriberBreach={exposureData} />
-  );
+  const [dataBrokerImage, setDataBrokerImage] = useState<ReactNode>();
+
+  useEffect(() => {
+    const loadDataBrokerImage = async () => {
+      if (isScanResult(exposureData)) {
+        const cleanedName = exposureData.data_broker.split(".")[0];
```

Comment:
It's probably a good idea to leave the TLD on, both to make it easier to find the website the logo was from, as well as to avoid problems if by any chance two instances share the same name but a different TLD (e.g. `truthfinder.com` vs `truthfinder.net`).